src/platform/forms/sass/_m-schemaform.scss

```diff
@@ -535,18 +535,14 @@ legend.schemaform-label.schemaform-file-label {
   }
 }
 
+/* Highlight accordion item with content that needs attention */
 va-accordion-item[data-unviewed-pages="true"] {
-  --item-border: 3px solid var(--color-secondary-lightest) !important;
-
   &::part(accordion-header) {
-    background-color: var(--color-secondary-lightest);
-    border-color: var(--color-secondary-lightest);
+    background-color: #{$color-secondary-lightest};
+  }
+  &::part(accordion-content) {
+    border-color: #{$color-secondary-lightest};
```

Comment:
Following "will work in all Sass versions" [recommendations from the documentation](https://sass-lang.com/documentation/breaking-changes/css-vars/)

Reply:
These changes, along with this [component library update](https://github.com/department-of-veterans-affairs/component-library/pull/1098), will fix the accordion highlighting on the review & submit page